config/healthcheck.php

```diff
@@ -13,16 +13,19 @@
      * of the service.
      */
     'checks' => [
-        OITNetworkServices\BuzzAPI\HealthChecks\CredentialsAreValid::class,
-        OITNetworkServices\BuzzAPI\HealthChecks\ServiceIsAvailable::class,
         UKFast\HealthCheck\Checks\CacheHealthCheck::class,
         UKFast\HealthCheck\Checks\DatabaseHealthCheck::class,
         UKFast\HealthCheck\Checks\HttpHealthCheck::class,
         UKFast\HealthCheck\Checks\LogHealthCheck::class,
         UKFast\HealthCheck\Checks\MigrationUpToDateHealthCheck::class,
-        UKFast\HealthCheck\Checks\RedisHealthCheck::class,
-        UKFast\HealthCheck\Checks\SchedulerHealthCheck::class,
         UKFast\HealthCheck\Checks\StorageHealthCheck::class,
+        ...('google-play-review' === env('APP_ENV') ? [
+            OITNetworkServices\BuzzAPI\HealthChecks\CredentialsAreValid::class,
```

Comment:
Why add the BuzzAPI checks only for the reviewer instance?

Reply:
BuzzAPI is flaky and I'm tired of getting spurious health check failures.

The Google Play review server, however, does not talk to real BuzzAPI, it hits a mock endpoint running on the same server, so I figured we may as well make sure that connectivity is working.